dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguagesResource.java

```diff
@@ -83,4 +180,26 @@ public Response getMessages(@Context HttpServletRequest request,
                                 final I18NForm i18NForm) {
         return oldLanguagesResource.getMessages(request, i18NForm);
     }
+
+    private Language saveOrUpdateLanguage(final String languageId, final LanguageForm form) {
+        final Language newLanguage = new Language();
+
+        if (StringUtils.isSet(languageId)) {
+            try {
+                final Language origLanguage = this.languageAPI.getLanguage(languageId);
+                BeanUtils.copyProperties(newLanguage, origLanguage);
+            } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4616952)